src/components/Book.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './Book.module.css';
+
+const Book = (props) => {
+  const { title, author } = props;
+
+  return (
+    <>
+      <div className={styles.book_wrapper}>
+        <div className={styles.content_wrapper}>
+          <div className={styles.content}>
+            <span className={styles.title}>{title}</span>
+            <span className={styles.author}>{author}</span>
+          </div>
+          <div className={styles.actions}>
+            <button type="button">Remove</button>
```

Comment:
- [ ] Please display the remove button. the button wouldn't remove the books yet, this is just a placeholder for now. Kindly follow the requirements.

Reply:
- [ ] Please, Just the way you displayed `Add New Book` from the component